### PR TITLE
refresh 05 writing a rust mutating policy

### PR DESCRIPTION
**Pull request by jhkrug**

### CODE REVIEW

File: docs/writing-policies/rust/05-mutation-policy.md [hunk 1]
```diff
@@ -51,11 +52,13 @@ spec:
 
 ## Write the mutation code
 
-The mutation code is done inside of the `validate` function. The function should be changed
-to approve the request via the [`mutate_request`](https://docs.rs/kubewarden-policy-sdk/0.1.0/kubewarden_policy_sdk/fn.mutate_request.html)
-instead of the [`accept_request`](https://docs.rs/kubewarden-policy-sdk/0.1.0/kubewarden_policy_sdk/fn.accept_request.html).
+The mutation code is in the `validate` function.
+You need to chnage the function to approve the request via the
```
Comment (jvanz) [inline docs/writing-policies/rust/05-mutation-policy.md:56]:
```suggestion
You need to change the function to approve the request via the
```

**Review by jvanz — CHANGES_REQUESTED** [review]
Besides the little typo, LGTM

File: docs/writing-policies/rust/05-mutation-policy.md (hunk 1 above)
Comment (jhkrug) [reply docs/writing-policies/rust/05-mutation-policy.md]: Thank you.

**Review by viccuad — APPROVED** [review]
LGTM!

**Comment by jhkrug** [comment]
Closing as #295 ended up having all the changes.

**Pull request closed without merging**